install_sct

```diff
@@ -514,13 +514,19 @@ done
 # Create directory
 mkdir -p "$SCT_DIR"
 # check if directory was created
+PERMISSION_ADVICE="\n\nMake sure that you own the folder you're writing to, and that the folder has write permissions.
+\nTo ensure this, either change the installation directory, or consider running the following commands:
+    sudo chown $USER $(dirname "$SCT_DIR")
+    sudo chown $USER $SCT_DIR
+    sudo chmod +w $(dirname "$SCT_DIR")
+    sudo chmod +w $SCT_DIR"
```

Comment:
Both `chown` and `chmod` accept a list of files, so the commands can be combined. And it would be much safer to scope the write permissions change to only the owning user:
```suggestion
    sudo chown $USER \"$(dirname "$SCT_DIR")\" \"$SCT_DIR\"
    sudo chmod u+w \"$(dirname "$SCT_DIR")\" \"$SCT_DIR\""
```

Reply:
I'm not sure if we need to escape the double-quotes? That seems to just result in the literal `"` character being part of the path, which ends up as an error for me:

```console
dot|joshua@mosaic:~$     sudo chmod u+w \"$(dirname "$SCT_DIR")\" \"$SCT_DIR\"
[sudo] password for joshua: 
chmod: cannot access '"/home/joshua/repos"': No such file or directory
chmod: cannot access '"/home/joshua/repos/spinalcordtoolbox"': No such file or directory
```

Apart from that, thank you for the tweaks to the command! <3